frontend/packages/pipelines-plugin/src/utils/render-utils.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { ExternalLink } from '@console/internal/components/utils';
+
+import './render-utils.scss';
+
+const URL_REGEXP = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&/=]*)/;
+export const GROUP_MATCH_REGEXP = new RegExp(`^(.*\\s)?(${URL_REGEXP.source})(\\s.*)?$`, 'i');
+
+export const handleURLs = (value: string): React.ReactNode => {
+  if (typeof value !== 'string') return value;
+
+  const matches = value.match(GROUP_MATCH_REGEXP);
+  const [, prefix, link, suffix] = matches || [];
+
+  if (link) {
+    return (
+      <>
+        {prefix}
+        <ExternalLink additionalClassName="opp-render-utils-external-url" href={link}>
+          {link}
+        </ExternalLink>
+        {suffix}
```

Comment:
Two things I noticed here
1. Suffix is incorrect
expected: 
`Visit http://example.com for logs`
Actual:
![image](https://user-images.githubusercontent.com/9964343/113283219-9d69ac80-9305-11eb-8860-fc2cd41914dd.png)

2. if there are multiple urls only one gets wrapped by external link other is a plain string
![image](https://user-images.githubusercontent.com/9964343/113284053-b1fa7480-9306-11eb-9dba-c25afd5a7451.png)

Note: There needs to be a space between the link and the suffix part.

Reply:
Yup, this is caused by my laziness lol. I used a fake RegExp to capture a URL, then googled for another one before I submitted. Re-ran my tests but didn't visually test it 🤦 

Thanks for the review 👍